.github/workflows/tests.yml

```diff
@@ -83,7 +83,7 @@ jobs:
           pip install -r requirements.txt
           pip install -e .
           pip freeze | grep torch
-          pytest -m fast --cov-fail-under 0
+          pytest -m fast --cov-fail-under 0 -n auto
```

Comment:
the hero that we needed

Reply:
Yes im shocked it works so well, i tried pytest-parallel and it was slow and buggy but pytest-xdist works like a dream!